doc_user/UserManual.adoc

```diff
@@ -801,6 +801,37 @@ image:image251.png[success]
 
 NOTE: This button can be enabled or disabled to control movements deletions. Ask the Administrator or check the Administrator's Guide for more information.
 
+===== 4.3.2.6 Adding Pagination and Search in MedicalBrowser ([.underline]##D##elete)
```

Comment:
Per the image the window title is "Pharmaceutical Browser" and not "MedicalBrowser".

What is up with the word "Delete" in the heading?

Use the same case for the image; in line 888 it has ".png" but image251 ends in ".PNG"